### DISCUSSION TITLE
aliyun和quark报错 请问是什么问题 如何解决 谢谢

### DISCUSSION THREAD

**Discussion by 52eraja** [AlistGo/alist#3942] [Q&A]
Post "https://open.aliyundrive.com/adrive/v1.0/user/getDriveInfo": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": net/http: TLS handshake timeout

**Comment by Learnmore-knowledge** [comment]
同问，我是用的replit 平台做服务器，电脑端挂的用的3.13.2版本就可以用，可能alist的原因吧

**Reply by Learnmore-knowledge** [reply]
我用其他服务器搭建目前为止都没出现问题，都是用的3.13.2版本，唯独replit出问题

**Comment by Mcayear** [comment]
难道夸克禁海外了？
